changelogs/fragments/71172-systemd-add-try-restarted-state.yml

```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
+  - systemd - Add try-restarted state (https://github.com/ansible/ansible/issues/61763)
```

Comment:
```suggestion
  - systemd - add ``try-restarted`` state (https://github.com/ansible/ansible/issues/61763).
```

Reply:
Fixed in https://github.com/ansible/ansible/pull/71172/commits/1fd4e67bcb1d16c1cbb64e57917f8de838511053.
Apologies for making the same mistake more than once:(

Reply:
No problem! This is a pretty common thing, and also you managed to commit everything at once (not everyone does ;) ).